lib/register/action.js

```diff
@@ -19,7 +19,7 @@ var chalk = require('chalk');
  * After you register a custom action, you then use that
  * action in a platform your config.json
  *
- * Actions run after the files in a platform are generated so you
+ * Actions run after the files in a platform are generated thus you
  * can perform operations on files generated by the style dictionary.
```

Comment:
You can perform operations on files generated by the style dictionary 
as actions run after the files in a platform are generated.